doc/TYPES2021/Demos/HSP.tex

```diff
@@ -466,38 +464,35 @@ \subsection{Logical foundations}
 \subsection{Setoids}\label{setoids}
 A \defn{setoid} is a pair consisting of a type \ab A and
 an equivalence relation \af{≈} on \ab A.  Setoids are useful for representing a
-set with a ``local'' notion of equivalence, instead of always relying on
-the global one as is usually done in set theory. In reality, informal
+set with a ``local'' notion of equivalence, instead of relying on
+the global one as is usual in set theory. In reality,
 mathematial practice uses equivalence relations quite pervasively, and
 takes great care to only define equivalence-preserving functions, while
-eliding the actual details. To be fully formal, such details must be given.
-While there are many different approaches for doing that, the one that requires
+eliding the actual details. To be properly formal, such details must be given.
+While there are many different workable approaches, the one that requires
 no additional meta-theory is based on setoids, which is why we adopt it here.
 While in some settings this has been found by others to be burdensome, we have not
 found it to be so for universal algebra.
 
-\ddmmyydate
+% \ddmmyydate
 
 The \agdaalgebras library was first developed without setoids, relying on
 propositional equality \ad{\au{}≡\au{}} instead,
 along with some experimental, domain-specific types for equivalence classes, quotients, etc.
-This furthermore required postulating function extensionality,%
+This required postulating function extensionality,%
 \footnote{An axiom that asserts that two point-wise equal functions are equal.} which is
 known to be independent from \mltt~\cite{MHE, MHE:2019}. This was
 an unsatisfactory state of affairs, as %we were curious to see if
 our aim is to show that the theorems hold directly in \mltt without extra axioms.
-In particular, the present exposition makes no appeals to classical axioms like Choice or Excluded Middle.
+In particular, we make no appeal to classical axioms like Choice or Excluded Middle.
 
 
 %% -----------------------------------------------------------------------------
 \subsection{Setoid functions}
 \label{setoid-functions}
-We use the \ar{Func} type from the \agdastdlib for representing a function from
-a setoid \ab A to another setoid \ab B that respects the underlying equivalences,
-witnessed by a field called \afld{cong}.  However, we rename \ar{Func}, using
-the more visually appealing infix long arrow symbol,
-\AgdaRecord{\AgdaUnderscore{}⟶\AgdaUnderscore{}}, instead. Throughout the
-paper we refer to inhabitants of this type as ``setoid functions.''
+A ``setoid function'' (denoted by
+\AgdaRecord{\AgdaUnderscore{}⟶\AgdaUnderscore{}}) is a function from
+a setoid \ab A to another setoid \ab B that respects the underlying equivalences.
```

Comment:
Okay... quite terse, but that's fine.  If the reader really cares to know about such details, they can look at the line renaming Func. :+1: